### PR TITLE
Project - Isabela Zeberio

### PR DESCRIPTION
**Pull request by isazeberio**

### CODE REVIEW

**Comment by isazeberio** [comment]
Hi Iñaki,
Happy new year!!
I think I'm almost done (I struggled quite a bit with GitHub). Anyways, I had a lot os issues with the size of the plots in R Markdown, and I didn't manage to set plots in a wider span that the margins. I am open to any changes you consider convenient to improve the visualization of the project.
Thank you in advance (sorry for the weird commits)

**Comment by Enchufa2** [comment]
Is this ready for review then?

**Comment by isazeberio** [comment]
Yes, it's ready for review.

File: _projects/2022/100386498/100386498.Rmd [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    self contained: false 
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(fig.showtext=TRUE)
+```
+
+![Diagram listing the 17 Sustainable Development Goals. Figure from [un.org](https://sdgs.un.org/goals)](/Users/isabelazeberio/Documents/GitHub/dataviz/_projects/2022/100386498/sustainabledevgoals.png)
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: All these paths should be relative to the Rmd, so it should be just 

```
![Diagram listing the 17 Sustainable Development Goals. Figure from [un.org](https://sdgs.un.org/goals)](sustainabledevgoals.png)
```

File: _projects/2022/100386498/100386498.Rmd [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+|17. **Partnership for the goals**                                                               |
+|Strengthen the means of implementation and revitalize the global partnership for sustainable development|                                                                                      
+:::
+
+The chart I have chosen is from the first SDGs: [No poverty](https://datatopics.worldbank.org/sdgatlas/archive/2017/SDG-01-no-poverty.html). In the section on **Reducing poverty in all its dimensions according to national definitions**, the figure 1c is introduced to represent how richer countries tend to have higher national poverty lines than poorer ones, and in some cases a richer country may even have a higher national poverty rate than a poorer country. 
+
+![The World Bank's Chart. Figure from [worldbank.org](https://datatopics.worldbank.org/sdgatlas/archive/2017/SDG-01-no-poverty.html)](/Users/isabelazeberio/Documents/GitHub/dataviz/_projects/2022/100386498/theworldbank.png)
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Use the relative path.

File: _projects/2022/100386498/100386498.Rmd [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+library(ggplot2)
+library(janitor)
+library(showtext)
+library(ggtext) 
+```
+
+I created the csv file coping the data from the Appendix 2 table. The variables needed to recreate the chart are the following:
+
+```{r include=FALSE}
+world_bank <- read_delim("/Users/isabelazeberio/Documents/GitHub/dataviz/_projects/2022/100386498/povertyline.csv",
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Use the relative path.

File: _projects/2022/100386498/100386498.Rmd [hunk 4]
```diff
@@ -0,0 +1,390 @@
...
+qntile %>% group_by(country_class) %>% 
+    summarise(country = n_distinct(country)) %>% 
+    arrange(desc(country))
+
+
+```
+
+Despite grouping the countries following the indications from the article by Jolliffe & Beer Prydz (2016), I obtained a slightly different classification. These differences mean that in my final chart some countries (dots) are colored differently than in the original chart.
+
+![Table 2: International Poverty Lines: Median and mean national lines, by group. Figure from [Jolliffe & Beer Prydz (2016)](https://datatopics.worldbank.org/sdgatlas/archive/2017/SDG-01-no-poverty.html)](/Users/isabelazeberio/Documents/GitHub/dataviz/_projects/2022/100386498/table2.png)
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Use the relative path.

File: _projects/2022/100386498/100386498.Rmd [hunk 5]
```diff
@@ -0,0 +1,390 @@
...
+  geom_point(data = transform(qntile,
+     country_class = NULL), colour = "grey80") + 
+  geom_point(alpha=0.7) +
+  ggrepel::geom_text_repel(aes(label=country)) +
+  scale_color_manual(values = c("#5F4690","#38A6A5", "#73AF48", "#E17C05")) +
+  facet_grid(~country_class, scales = "free") + 
+  theme_bw()
+```
+
+```{r echo=FALSE, fig.width=15,fig.height=12,fig.align = "center"}
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Please do not hide any chunk. Remove the `echo=FALSE`. In fact, you could merge this chunk with the previous one.

File: _projects/2022/100386498/100386498.Rmd [hunk 6]
```diff
@@ -0,0 +1,390 @@
...
+   ggplot( aes(x=HFCE, y=national_povt_line, color = country_class, text=text)) +
+   geom_point(alpha=0.7) +
+   scale_size(range = c(1.4, 19), name="Dollars ($)") +
+   scale_color_manual(values = c("#5F4690","#38A6A5", "#73AF48", "#E17C05")) +
+   theme(legend.position="none")
+
+```
+
+
+```{r include=FALSE}
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Please do not hide any chunk. Remove the `include=FALSE`.

File: _projects/2022/100386498/100386498.Rmd [hunk 7]
```diff
@@ -0,0 +1,390 @@
...
+p2 <- p2 +
+  scale_x_log10() +
+  scale_y_log10() +
+  theme_minimal() 
+```
+
+
+```{r include=FALSE}
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Please do not hide any chunk. Remove the `include=FALSE`.

File: _projects/2022/100386498/100386498.Rmd [hunk 8]
```diff
@@ -0,0 +1,390 @@
...
+library(tidyverse)
+library(ggplot2)
+library(janitor)
+library(showtext)
+library(ggtext) 
+```
+
+I created the csv file coping the data from the Appendix 2 table. The variables needed to recreate the chart are the following:
+
+```{r include=FALSE}
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Please do not hide any chunk. Remove the `include=FALSE`.

File: _projects/2022/100386498/100386498.Rmd [hunk 9]
```diff
@@ -0,0 +1,390 @@
...
+    arrange(desc(country))
+
+
+```
+
+Despite grouping the countries following the indications from the article by Jolliffe & Beer Prydz (2016), I obtained a slightly different classification. These differences mean that in my final chart some countries (dots) are colored differently than in the original chart.
+
+![Table 2: International Poverty Lines: Median and mean national lines, by group. Figure from [Jolliffe & Beer Prydz (2016)](https://datatopics.worldbank.org/sdgatlas/archive/2017/SDG-01-no-poverty.html)](/Users/isabelazeberio/Documents/GitHub/dataviz/_projects/2022/100386498/table2.png)
+
+```{r include=FALSE}
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Please do not hide any chunk. Remove the `include=FALSE`.

File: _projects/2022/100386498/100386498.Rmd (hunk 1 above)
Comment (Enchufa2) [reply _projects/2022/100386498/100386498.Rmd]: Please, also, add `{.external width="100%"}` to the end of the line.

File: _projects/2022/100386498/100386498.Rmd (hunk 2 above)
Comment (Enchufa2) [reply _projects/2022/100386498/100386498.Rmd]: Please, also, add `{.external width="100%"}` to the end of the line.

File: _projects/2022/100386498/100386498.Rmd (hunk 5 above)
Comment (Enchufa2) [reply _projects/2022/100386498/100386498.Rmd]: Additionally, I think this plot would look nicer with something like `fig.asp="0.3"`  (after removing the current `fig.height`) and `layout="l-page"` or even `layout="l-screen"`. You can play with those values.

File: _projects/2022/100386498/100386498.Rmd [hunk 10]
```diff
@@ -0,0 +1,390 @@
...
+### Interactive chart
+Another alternative visualization is to plot an interactive scatterplot by using `ploytly`. This way more information is added to every dot without increasing data density:
+
+  - Country
+  - Year
+  - National poverty line
+  - Household final consumption
+  - GNI
+
+```{r fig.width=100,fig.height=90,fig.align = "center"}
```
Comment (Enchufa2) [inline _projects/2022/100386498/100386498.Rmd]: Currently, this plot doesn't fit into my screen vertically. A `fig.asp=0.7`  (after removing the current `fig.height`) may look better.

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Great work! Looks very nice. Just a few comments below. Also, there are some lines of code that are still too wide. It would be nice to wrap them in 80 characters max.

**Comment by isazeberio** [comment]
Hi Iñaki,
I changed the details you mentioned in your review, thank you very much!! The only thing I am not very happy with my work is the dimensions of the graph with the 4 facets. I haven't been able to match layout="l-page" with ggrepel very well.

**Comment by Enchufa2** [comment]
Great, but you need to commit and push all the changes! :) Meaning... add the whole `_projects` folder, commit and push it.

**Review by Enchufa2 — APPROVED** [review]
Great job, thanks! I've included a few minor tweaks (the `fig.asp` must be numeric ;-) ) that you can get by clicking **Pull** in RStudio. Merging!

**Comment by Enchufa2** [comment]
And here you are: https://csslab.uc3m.es/dataviz/projects/2022/100386498/

**Pull request merged**